src/stars.css

```diff
@@ -0,0 +1,41 @@
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+/* .stars + .stars {
+  margin-top: 4px;
+} */
+
+.stars--1 :nth-child(-n+1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
```suggestion

```

Reply:
my new commit doesn't update the job status, it's still considered rejected even though I updated it